src/services/identity/__tests__/AuthService.spec.ts

```diff
@@ -57,11 +51,8 @@ describe("Auth Service", () => {
 
   it("should call axios successfully and bubble the error when the status is not 403 or 404", async () => {
     // Arrange
-    const expected = {
-      response: { status: "400" },
-    }
     mockGitHubService.checkHasAccess.mockRejectedValueOnce(
-      new BadRequestError(expected)
+      new BadRequestError("")
```

Comment:
could I get context on why this returns a different shape now?

Reply:
it still accepts the same shape - last time no type safety cos in `js` so it accepts object, but `message` is a `string`